Vendor/iOS/MSAlertController/MSAlertController.m

```diff
@@ -167,7 +168,11 @@ + (void)makeKeyAndVisible {
     if (!windowScene) {
       windowScene = scenes.anyObject;
     }
-    MS_DISPATCH_SELECTOR(void, window, setWindowScene:, windowScene);
+    
+    // FIXME: Explicit typing to call `setWindowScene:` as windowScene maybe nil and the macro won't support it.
+    SEL selector = NSSelectorFromString(@"setWindowScene:");
+    IMP impl = [window methodForSelector:selector];
+    ((void (*)(id, SEL, typeof(window))) impl)(window, selector, windowScene);
```

Comment:
typeof(windowScene)?

Reply:
There is a FIXME comment that we can address it later. This PR isn't a fix for that but master to develop merge.